packages/terminal/src/browser/terminal-frontend-contribution.ts

```diff
@@ -84,10 +84,9 @@ export class TerminalFrontendContribution implements CommandContribution, MenuCo
         /* Register ctrl + k (the passed Key) as a passthrough command in the
            context of the terminal.  */
         const regCtrl = (k: Key) => {
-            const modifier = isOSX ? Modifier.M4 : Modifier.M1;
             keybindings.registerKeybindings({
                 commandId: KeybindingRegistry.PASSTHROUGH_PSEUDO_COMMAND,
-                keyCode: KeyCode.createKeyCode({ first: k, modifiers: [modifier] }),
+                keyCode: KeyCode.createKeyCode({ first: k, modifiers: [Modifier.CTRL] }),
```

Comment:
why is that changed?

Reply:
I extracted the check `isOSX ? Modifier.M4 : Modifier.M1` to a new constant that can (should) be used where CTRL is needed independently of the platform. The behavior is unchanged.

Reply:
OK indeed nice :)